_posts/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-04-plot-data-matplotlib.md

```diff
@@ -0,0 +1,256 @@
+
+
+
+
+## Customize Your Plot
+
+### Add Title and Axis Labels
+
+Expand your code to add a title to the plot and to label the axes. 
+
+{:.input}
+```python
+# set plot size for all plots that follow
+plt.rcParams["figure.figsize"] = (8, 8)
+
+# set plot title size for all plots that follow
+plt.rcParams['axes.titlesize'] = 20
+
+# create the plot space upon which to plot the data
+fig, ax = plt.subplots()
+
+# add the x-axis and the y-axis to the plot
+ax.plot(months, precip)
+
+# set plot title
+ax.set(title="Average Monthly Precipitation in Boulder, CO")
+
+# add labels to the axes
+ax.set(xlabel="Month", ylabel="Precipitation (mm)");
+```
+
+{:.output}
+{:.display_data}
+
+![png]({{ site.url }}//images/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-04-plot-data-matplotlib_8_0.png)
+
+
+
+
+## Change Plot Type
+
+You can turn your plot into a bar plot using `ax.bar()`, providing the x- and y-axes as you did with `ax.plot()`.
+
+You can also assign a fill color using `color="green"`. 
+
+{:.input}
+```python
+# set plot size for all plots that follow
+plt.rcParams["figure.figsize"] = (8, 8)
+
+# create the plot space upon which to plot the data
+fig, ax = plt.subplots()
+
+# add the x-axis and the y-axis to the plot
+ax.bar(months, precip, color="green")
+
+# set plot title
+ax.set(title="Average Monthly Precipitation in Boulder, CO")
+
+# add labels to the axes
+ax.set(xlabel="Month", ylabel="Precipitation (mm)");
+```
+
+{:.output}
+{:.display_data}
+
+![png]({{ site.url }}//images/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-04-plot-data-matplotlib_10_0.png)
+
+
+
+
+**Congratulations - you have created your first customized plots of data!** 
+
+<div class="notice--warning" markdown="1">
+
+## <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Optional Challenge 
+
+Test your `Python` skills to further customize your plot:
+
+1. Recreate the x-axis to use full month name (e.g. `January`, not `Jan`) (hint: create a new `months` list). 
+
+2. Rotate the x-axis markers using `plt.setp(ax.get_xticklabels(), rotation=45)`, so that the spacing along the x-axis is more appealing now that the months are longer. 
+
+3. Change your plot to a scatter plot using `ax.scatter()` (hint: see `ax.bar(months, precip)`).
+
+</div>
+
+
+
+{:.output}
+{:.display_data}
+
+![png]({{ site.url }}//images/courses/earth-analytics-bootcamp/02-python-variables-lists/exercises/2018-07-23-python-variables-lists-04-plot-data-matplotlib_12_0.png)
+
+
+
+
+<div class="notice--info" markdown="1">
+
+## <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Additional Resources
+
+<a href="https://matplotlib.org/examples/pylab_examples/subplots_demo.html" target="_blank">More Matplotlib Examples</a>
```

Comment:
This is good. please just list these as a list using 
`* `
so any additional resources box will look like this:

```
* <a href="https://matplotlib.org/examples/pylab_examples/subplots_demo.html" target="_blank">More Matplotlib Examples</a>
* <a href="https://realpython.com/blog/python/python-matplotlib-guide/?utm_campaign=Data%2BElixir&utm_medium=email&utm_source=Data_Elixir_172" target="_blank">In-depth Guide to Matplotlib</a>
* <a href="http://joseph-long.com/writing/colorbars/" target="_blank">More Information on Color Bars</a>
	
```